print-nfs.c

```diff
@@ -1435,15 +1435,17 @@ parsecreateopres(netdissect_options *ndo,
 	return (NULL);
 }
 
-static int
+static const uint32_t *
 parsewccres(netdissect_options *ndo,
             const uint32_t *dp, int verbose)
 {
 	u_int er;
 
 	if (!(dp = parsestatus(ndo, dp, &er)))
 		return (0);
-	return parse_wcc_data(ndo, dp, verbose) != NULL;
+	if (!(dp = parse_wcc_data(ndo, dp, verbose)))
+		return (0);
+	return dp;
```

Comment:
this is the same as `return parse_wcc_data(ndo, dp, verbose)`

Reply:
Yup, that is true, I can make it simpler.